nncf/quantization/advanced_parameters.py

```diff
@@ -111,6 +111,25 @@ class AdvancedBiasCorrectionParameters:
     threshold: Optional[float] = None
 
 
+@api()
+@dataclass
+class AdvancedSmoothQuantParameters:
+    """
+    Contains advanced alpha parameters for SmoothQuant algorithm.
+    It regulates the calculation of the smooth scale for different node types.
+    A negative value switches off the algorithm for current node type. In case of inaccurate results,
+    this parameter may be adjusted in the range from 0 to 1 or set -1 to disable SmoothQuant algorithm.
+
+    :param convolution: Whether to apply smoothing for Convolution layers.
+    :type convolution: float
+    :param matmul: Whether to apply smoothing for MatMul layers.
+    :type matmul: float
+    """
+
+    convolution: float = -1
+    matmul: float = 0.95
```

Comment:
```suggestion
    convolution_alpha: float = -1
    matmul_alpha: float = 0.95
```

Reply:
Wouldn't be better to rename `smooth_quant_params` into `smooth_quant_alphas` and keep these arguments as is?

Reply:
Sounds good.

Reply:
Done